### ISSUE TITLE
frontend: add new user account for DIdentity

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#42]
## User story
1. As a {Admin of DIdentity}
2. I want / need {to create a user account for new HR-employees}
3. So that {the new HR-employee can do his/her work}

## Acceptance criteria
* page with fields:
- name
- surname
- email-address
- role of new account (another admin or HR-employee)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
